packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-content.tsx

```diff
@@ -9,24 +9,42 @@ const JurisdictionModalContent = ({
     setJurisdictionSelectedShortcode,
     synthetic_available_accounts,
     financial_available_accounts,
+    swapfree_available_accounts,
     context,
     real_synthetic_accounts_existing_data,
     real_financial_accounts_existing_data,
+    real_swapfree_accounts_existing_data,
 }: TJurisdictionModalContentProps) => {
     const card_classname = `cfd-jurisdiction-card--${account_type}`;
 
-    const cardsToBeShown = (type_of_card: string) =>
-        account_type === 'synthetic'
-            ? synthetic_available_accounts?.some(account => account.shortcode === type_of_card)
-            : financial_available_accounts?.some(account => account.shortcode === type_of_card);
-
+    const cardsToBeShown = (type_of_card: string) => {
+        switch (account_type) {
+            case 'synthetic':
+                return synthetic_available_accounts?.some(account => account.shortcode === type_of_card);
+            case 'all':
+                return swapfree_available_accounts?.some(account => account.shortcode === type_of_card);
+            default:
+                return financial_available_accounts?.some(account => account.shortcode === type_of_card);
+        }
+    };
```

Comment:
Same with the below comment

Reply:
the thing is it is passed down to other component so cant do it for now